README.MD

```diff
@@ -1966,7 +1966,27 @@ Properties:
 
 Predefined expandable properties:
 
-* `instance` - [LocalInstance](src/main/kotlin/com/cognifide/gradle/aem/common/instance/LocalInstance.kt) object.
+* `instance` - [LocalInstanceManager](src/main/kotlin/com/cognifide/gradle/aem/common/instance/LocalInstanceManager.kt) object.
```

Comment:
that's wrong! in files being expanded there is var named `instance` of type `LocalInstance`